statsmodels/tsa/exponential_smoothing/_ets_smooth.pyx

```diff
@@ -0,0 +1,153 @@
+#!python
+#cython: wraparound=False, boundscheck=False, cdivision=True, annotate=True
+
+import numpy as np
+cimport numpy as np
+
+np.import_array()
+
+cpdef _initialize_ets_smooth(np.float_t [:] params, Py_ssize_t n):
```

Comment:
We should use a fused type that supports both float32 and float64 here?  This would produce "free" version that let users use single or double precision data.

Reply:
https://cython.readthedocs.io/en/latest/src/userguide/fusedtypes.html

Reply:
Done